### PR TITLE
Add explanation for cut command

### PR DESCRIPTION
**Pull request by hannahburkhardt**
This fixes issue #608 using the text suggested there (with minor modification)

### CODE REVIEW

**Comment by gdevenyi** [comment]
Thanks for the contribution, I have a few suggestions for changes and then I think we're good to go!

**Comment by hannahburkhardt** [comment]
I can't seem to see anywhere what your suggestions are - how can I view them?

**Comment by gdevenyi** [comment]
Ooops sorry! I always forgot I have to push "request changes" for my review to show up!

File: _episodes/04-pipefilter.md [hunk 1]
```diff
@@ -549,14 +549,15 @@ so that you and other people can put those programs into pipes to multiply their
...
-> For the file `animals.txt` from the previous exercise, the command:
+> For the file `animals.txt` from the previous exercise, consider the following command:
 >
 > ~~~
 > $ cut -d , -f 2 animals.txt
 > ~~~
 > {: .language-bash}
 >
-> uses the `-d` option to split each line by comma, and the `-f` option
+> The `cut` command is used to remove or "cut out" certain sections of each line in the file. The optional `-d` flag is used to define the delimiter. The default delimiter is tab. The `-f` flag is used to specify the field to cut out.
```
[the remark this answers is not in the document]
Comment (hannahburkhardt) [reply _episodes/04-pipefilter.md]: Done! Thanks for your feedback.

**Comment by gdevenyi** [comment]
Thanks!

**Pull request merged**